drivers/docker/network.go

```diff
@@ -0,0 +1,84 @@
+package docker
+
+import (
+	"fmt"
+
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/hashicorp/nomad/plugins/drivers"
+)
+
+const infraContainerImage = "gcr.io/google_containers/pause-amd64:3.0"
```

Comment:
is this configurable somewhere? We don't really want our infra to reach out to foreign container registries at any time - we mirror _everything_ into ECR in the local AWS region

Reply:
Yeah I am going to get that plumbed up into a client config option.

Reply:
Yeah I am going to get that plumbed up into a client config option.